bin/commands/test/execute-on-demand.js

```diff
@@ -17,25 +17,39 @@ module.exports = {
   },
 
   // TODO; what happens when we execute with multiple browsers, etc?
```

Comment:
Is this still a question or has it been addressed?

Reply:
No, I missed this endpoint! I'm going to address this today

Reply:
This has been addressed in https://github.com/ghost-inspector/node-ghost-inspector/pull/35, it should be merged down before we release this.